src/Sulu/Component/Content/Metadata/Loader/XmlLegacyLoader.php

```diff
@@ -82,7 +82,12 @@ public function load($resource, $type = 'page')
         $schemaPath = __DIR__ . static::SCHEME_PATH;
 
         // read file
-        $xmlDocument = XmlUtils::loadFile($resource, $schemaPath);
+        $xmlDocument = XmlUtils::loadFile($resource, function (\DOMDocument $dom) use ($resource, $schemaPath) {
+                $dom->documentURI = $resource;
+                $dom->xinclude();
+
+                return @$dom->schemaValidate($schemaPath);
```

Comment:
The errors get handled afterwards in XmlUtils::getXmlErrors, which are more detailed according to a [comment](http://php.net/manual/en/domdocument.schemavalidate.php#62032) on the php documentation.